### ISSUE TITLE
perl-finance-bank-laposte stop working since 02/07/2016

### ISSUE THREAD

**Issue by testcocoon** [pixel/perl-finance-bank-laposte#5]
I guess it is a problem of the URL on La Banque Postale which changed.

Error message:
Use of uninitialized value $rel in pattern match (m//) at /usr/local/share/perl/5.20.2/Finance/Bank/LaPoste.pm line 161.
Use of uninitialized value $rel in concatenation (.) or string at /usr/local/share/perl/5.20.2/Finance/Bank/LaPoste.pm line 165.
getting https://voscomptesenligne.labanquepostale.fr/voscomptes/canalXHTML/identif.ea/../ failed
<html>
<head><title>An Error Occurred</title></head>
<body>
<h1>An Error Occurred</h1>
<p>500 Internal Server Error</p>
</body>
</html>

**Comment by prigaux** [comment]
Can you try this change:

```
--- a/lib/Finance/Bank/LaPoste.pm
+++ b/lib/Finance/Bank/LaPoste.pm
@@ -261,7 +261,7 @@ sub new {
     exists $self->{password} or croak "Must provide a password";
     exists $self->{username} or croak "Must provide a username";
 
-    $self->{ua} ||= LWP::UserAgent->new;
+    $self->{ua} ||= LWP::UserAgent->new(agent => '');
 
     _login($self);
     $self;
```

**Comment by testcocoon** [comment]
Thanks. It fix the issue.

**Comment by testcocoon** [comment]
It works partially.
In fact the globan amount can be retrieved with this fix.
But I do not get the list of transactions.

Also there are few warnings (I'm not sure if this could be the reason for the issue):
Use of uninitialized value $balance in substitution (s///) at /usr/local/share/perl/5.20.2/Finance/Bank/LaPoste.pm line 359.  
Use of uninitialized value $solde_month in string eq at /usr/local/share/perl/5.20.2/Finance/Bank/LaPoste.pm line 371. 
Use of uninitialized value $solde_month in string eq at /usr/local/share/perl/5.20.2/Finance/Bank/LaPoste.pm line 371.

**Comment by prigaux** [comment]
Reproduced.
Better:
```
LWP::UserAgent->new(agent => 'Mozilla');
```
works for me

**Comment by prigaux** [comment]
So be it: I released new release (7.10) with that fix.
